django/src/rdwatch/models/hyper_parameters.py

```diff
@@ -13,6 +16,11 @@ class HyperParameters(models.Model):
         help_text='The hyper parameters for an ML task',
         db_index=True,
     )
+    expiration_time = models.DurationField(
+        null=True,
+        blank=True,
+        help_text='Time relative to creation that this model run should be deleted.',
+    )
```

Comment:
While this is null/blank.  I think the POST endpoint requires that there is an `expiration_time`.  Might want to sync these up, or come up with some reasonable default time?

Reply:
Yeah that was an oversight, we do want to allow null/blank here because not all model runs should have an expiration date (Ground truths for example, we never want deleted)